src/components/Chart/BarChart.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import C3Chart from 'react-c3js';
+import { getComposer } from './constants';
+
+const BarChart = getComposer('BAR_CHART')(
+  ({ className, type, data, ...props }) => (
+    <C3Chart className={className} type={type} data={data} {...props} />
```

Comment:
should this not have a way to set the bar chart title on hover? I currently see: 
![recorded](https://user-images.githubusercontent.com/20167/33937237-589ca162-e00b-11e7-84e6-98ce375518b8.gif)

Reply:
Fixed 👍